pygmt/src/binstats.py

```diff
@@ -84,7 +84,7 @@ def binstats(data, **kwargs):
         Normalize the resulting grid values by the area represented by the
         search *radius* [no normalization].
     search_radius : float or str
-        Sets the *search_radius* that determines which data points are
+        Set the *search_radius* that determines which data points are
         considered close to a node. Append the distance unit.
         Not compatible with ``tiling``.
```

Comment:
At the moment ``tiling`` is not added as an alias  for **T**  (https://docs.generic-mapping-tools.org/latest/gmtbinstats.html#t).

Reply:
I submitted a PR to add this alias to the documentation of `pygmt.binstats`, please see PR #2409.